extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -1198,6 +1198,12 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
               'Attempt to change size failed on fluid ' +
                 'creative. Will re-attempt when slot is out of the viewport.'
             );
+            const {width, height} = this.getSlotSize();
+            if (width > 0 && height > 0) {
```

Comment:
Nit: > 0 here is unnecessary esp since you know this returns a number.  Could make this if (width * height) { ... }

Reply:
Went with width && height here and elsewhere. LMK if there was a particular reason for doing width * height.